app/models.py

```diff
@@ -419,6 +419,19 @@ def __repr__(self):
         return '<MatomoDailyGetSiteSearchKeywords {}>'.format(self.id)
 
 
+class ArkId(db.Model):
```

Comment:
I did not go down that road because the pipeline table is actually empty. Instead, the portal is generating a summary boutiques JSON file that displays the pipeline information. Therefore, I created an ark_id table so that all ARK IDs are stored in one place with the links to the Zenodo ID for the pipelines and to the dataset ID of the dataset table.

Does that make sense?